lib/controller/EetplanController.class.php

```diff
@@ -114,7 +114,15 @@ public function bekendehuizen($actie = null) {
             } else {
                 $this->view = $form;
             }
-
+        } elseif ($actie == 'verwijderen') {
+            $selection = filter_input(INPUT_POST, 'DataTableSelection', FILTER_SANITIZE_STRING, FILTER_FORCE_ARRAY);
+            $verwijderd = array();
+            foreach ($selection as $uuid) {
+                $eetplan = $this->model->getUUID($uuid);

```

Comment:
> Om een klein beetje van de ontzettende singleton-heid van modellen af te stappen.

Dit is conceptueel zo gekozen omdat je ook maar 1 instantie hebt van een tabel in de database en je niet meerdere modellen wilt hebben voor data-consistentie. Of je moet alles omschrijven naar transacties.